pom.xml

```diff
@@ -130,6 +130,14 @@
             </modules>
         </profile>
 
+        <profile>
+            <id>thin</id>
```

Comment:
Let's rename to "platforms" to match the directory name.

Reply:
Fixed.